src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import cn from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+export const Navigation = () => {
+  return (
+    <nav
+      data-cy="nav"
+      className="navbar is-fixed-top has-shadow"
+      role="navigation"
+      aria-label="main navigation"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink
+            to={'/'}
+            className={({ isActive }) =>
+              cn('navbar-item', { 'has-background-grey-lighter': isActive })
+            }
+          >
+            Home
+          </NavLink>
+
+          <NavLink
+            to={'/people'}
```

Comment:
```suggestion
            to="/people"
```
You don't braces if you just pass a string